src/pages/document/[id].tsx

```diff
@@ -451,7 +424,44 @@ const FamilyPage: InferGetServerSidePropsType<typeof getServerSideProps> = ({
               </section>
             ))}
           </SingleCol>
-        </SiteWidth>
+          {/* TODO: use a panel for this */}
+          {concepts.length > 0 && (
+            <div className="grow-0 shrink-0 px-5 border-l pt-5 w-[460px] text-sm">
```

Comment:
Question: What happens here on mobile? I don't see anything related to responsiveness.

Reply:
From @conordelahunty's (and my) thoughts: "that’s beyond our remit right now".
The point of this is to get it to internal people as soon as.

As a minimum - would hiding it be OK?

Reply:
I had forgotten we have featured flags for this kind of quick iteration, so ignore me. I think its fine to circle back later.